### PR TITLE
feat(breakpoints): added new SDS breakpoints

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**SDS Theme**
Github issue: #800

## Checklist

- [ ] Default Story in Storybook
- [ ] LivePreview Story in Storybook
- [ ] Test Story in Storybook
- [ ] Tests written
- [ ] Variables from `defaultTheme.ts` used wherever possible
- [ ] If updating an existing component, depreciate flag has been used where necessary
- [ ] Chromatic build verified by @chanzuckerberg/sds-design

### CODE REVIEW

File: packages/components/src/core/styles/common/types.ts [hunk 1]
```diff
@@ -97,6 +97,20 @@ declare module "@mui/material/styles" {
...
     sds?: SDSPalette;
   }
+
+  /**
+   * @see https://mui.com/material-ui/customization/breakpoints/#custom-breakpoints
+   * (masoudmanson): ModuleAugmentation is required for TypeScript to recognize the
+   * custom breakpoints added to the theme.
+   */
+  export interface BreakpointOverrides {
+    // remove the xs and xl breakpoints since SDS does not support them
```
Comment (tihuan) [inline packages/components/src/core/styles/common/types.ts:107]: awesome comment! Thank you 👏

**Review by tihuan — APPROVED** [review]
Really exciting 🤩 We got breakpoints now ⚡ ⚡ ⚡ Thanks so much for adding this, @masoudmanson !!

File: packages/components/src/core/styles/common/types.ts (hunk 1 above)
Comment (masoudmanson) [reply packages/components/src/core/styles/common/types.ts:107]: Thanks so much for taking the time to go through the PRs! Really grateful for your time and feedback! You're amazing 😍🥳🎉✨

Comment (tihuan) [reply packages/components/src/core/styles/common/types.ts:107]: Always a pleasure, Masoud!! Thank you again for your continued outstanding contribution to SDS 🤩 🔥 ✨🙏 👏  !!

**Review by clarsen-czi — APPROVED** [review]
LGTM!! Thanks @masoudmanson!

**Pull request merged**